canonical/ubuntu.com
# Update align-vertically instances

# Done

Remove all instances of row-image-centered and replace with equal height and equal-height__align-vertically. Also remove the newer .align-vertically (used with .equal-height) and replace with equal-height__align-vertically.
## QA

Requires cutting edge vanilla-framework and ubuntu-vanilla-theme and a gulp build of each, you may need to save styles.css in ubuntu-website to give the compiler a kick. Check all instances of equal-height__align-vertically to make sure they’re aligned vertically.
## Details

Card: https://trello.com/c/2VV18QqF
